src/analyze/analyze-chid.c

```diff
@@ -90,10 +113,16 @@ static int add_chid(Table *table, const EFI_GUID guids[static CHID_TYPES_MAX], s
         if (!flags)
                 return log_oom();
 
-        r = table_add_many(table,
-                           TABLE_UINT, (unsigned) t,
-                           TABLE_STRING, flags,
-                           TABLE_UUID, id);
+        if (t < EXTRA_CHID_BASE)
+                r = table_add_many(table,
+                                   TABLE_UINT, (unsigned) t,
+                                   TABLE_STRING, flags,
+                                   TABLE_UUID, id);
+        else
+                r = table_add_many(table,
+                                   TABLE_STRING, extra_chid_types[t - EXTRA_CHID_BASE],
```

Comment:
please get rid of the extra_chid_types table. just do `table_add_cell_stringf()`